### ISSUE TITLE
A plain YAML file for installation into new clusters

### ISSUE THREAD

**Issue by max-rocket-internet** [aws/amazon-eks-pod-identity-webhook#10]
Can we have a plain YAML file for installation, like in the AWS CNI repo?

For example, so we could do something like this:

```
kubectl apply -f https://raw.githubusercontent.com/aws/amazon-eks-pod-identity-webhook/release-1.0/config/v1.0/config.yaml
```

**Comment by micahhausler** [comment]
On second thought, I'm not sure this is possible, due to the CA needing to be [embedded in the webhook](https://github.com/aws/amazon-eks-pod-identity-webhook/blob/master/deploy/mutatingwebhook.yaml#L14)

**Comment by max-rocket-internet** [comment]
Yeah we'll use the new chart when docker images are finally available and not a `kubectl apply -f xx` method. But people will still want some alternative to Helm I guess. Feel free to close this issue.

**Comment by anguslees** [comment]
> On second thought, I'm not sure this is possible, due to the CA needing to be embedded in the webhook

I encourage you to re-open https://github.com/kubernetes/kubernetes/issues/60690 :)
